src/components/views/settings/tabs/user/SessionManagerTab.tsx

```diff
@@ -38,11 +40,24 @@ const SessionManagerTab: React.FC = () => {
         }
     };
 
+    const onGoToFilteredList = (filter: DeviceSecurityVariation) => {
+        setFilter(filter);
+        // @TODO(kerrya) clear selection when added in PSG-659
+        clearTimeout(scrollIntoViewTimeoutRef.current);
+        // wait a tick for the filtered section to rerender with different height
+        scrollIntoViewTimeoutRef.current =
+            window.setTimeout(() => filteredDeviceListRef.current?.scrollIntoView(true));
```

Comment:
Should we maybe specify `behavior=smooth` for a slightly nicer feeling experience?